package.json

```diff
@@ -16,29 +16,29 @@
   ],
   "dependencies": {
     "@smartthings/smartapp": "^1.0.3",
-    "body-parser": "*",
+    "body-parser": "^1.19.0",
```

Comment:
I actually just noticed in my local repo that body parser was crossed out cause it's apparently deprecated, is that recent or did I just not see it before?

Reply:
yeah we used it in school recently as well as here, and it's never been crossed out before so I thought it was weird too!